wg-arch-s390x/charter.md

```diff
@@ -0,0 +1,31 @@
+# WG S390x architecture Charter
+
+## Scope
+
+The S390x architecture working group takes care of all things related to supporting s390x architecture on KubeVirt clusters.
+This includes all build processes, also support of failing builds or test lanes.
+
+### In scope
+
+#### Code, Binaries and Services
+
+- list of what qualifies a piece of code, binary or service
```

Comment:
@cfilleke @jschintag @vamsikrishna-siddu hey, here's the charter for the wg arch S390x, please chime in here with the things you want to have added, I'll happily integrate them

Reply:
Thanks for getting this party started, @dhiller !

Reply:
Hi @dhiller I have put the things I want to add for s390x in my branch here. https://github.com/vamsikrishna-siddu/community/commit/b7c6af2d8f128895e961ec3e40897731d9821424 
can you please review them and add those changes in your PR for s390x.
Thanks